code/src/App.js

```diff
@@ -1,9 +1,48 @@
-import React from 'react'
+import React, {useState} from 'react';
+import "./index.css";
+import "./app.css";
+import {Summary} from "./Summary";
+import {NameQuestion} from "./NameQuestion";
+import {AnimalQuestion} from "./AnimalQuestion";
+import {SureQuestion} from "./SureQuestion";
+import {OtherAnimalQuestion} from "./OtherAnimalQuestion"
 
```

Comment:
I have just learnt from Maskymilian this:

import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';

So, final order is :

Outer dependencies
Local dependencies
Style file(s)

Reply:
👍